doc/source/robot/example.robot

```diff
@@ -9,7 +9,7 @@ ${MESSAGE}     Hello,
 ...    world!
 
 *** Test Cases ***
-First Test
+First, Test
```

Comment:
I would rather have specific faulty test cases, just to keep the example more clean. This now implies that you can add commas to the test case title, but I do not want to make that as a standard assumption.

Reply:
937d6020767d6a6c368880262a694f10866c75ca